document/html/index.bs

```diff
@@ -0,0 +1,389 @@
+    text: alloc_global; url: appendix/embedding.html#embed-alloc-global
+    text: type_global; url: appendix/embedding.html#embed-type-global
+    text: read_global; url: appendix/embedding.html#embed-read-global
+    text: write_global; url: appendix/embedding.html#embed-write-global
+    text: module_exports; url: appendix/embedding.html#embed-exports
+    text: error; url: appendix/embedding.html#embed-error
+    text: store; url: exec/runtime.html#syntax-store
+    text: table type; url: syntax/types.html#syntax-tabletype
+    text: table address; url: exec/runtime.html#syntax-tableaddr
+    text: function address; url: exec/runtime.html#syntax-funcaddr
+    text: memory address; url: exec/runtime.html#syntax-memaddr
+    text: i64; url: syntax/types.html#syntax-valtype
+    text: function element; url: exec/runtime.html#syntax-funcelem
+    text: import component; url: syntax/modules.html#imports
+    text: external value; url: exec/runtime.html#syntax-externval
+    text: host function; url: runtime.html#syntax-hostfunc
+    text: the instantiation algorithm; url: exec/modules.html#instantiation
+urlPrefix: https://littledan.github.io/spec/document/JS.html; spec: WASMJS
+    type: namespace
+        text: WebAssembly; url: #namespacedef-webassembly
+        text: CompileError; url: #compileerror
+        text: Module; url: #module
+        text: WebAssemblyInstantiatedSource; url: #dictdef-webassemblyinstantiatedsource
+        text: InstanceExportsMap; url: #typedefdef-instanceexportsmap
+        text: InstanceImportsMap; url: #typedefdef-instanceimportsmap
+        text: ExportedFunction; url: #exportedfunction
+    type: dfn
+        text: compile a WebAssembly module; url: #compile-a-webassembly-module
+        text: instantiate a WebAssembly module; url: #instantiate-a-webassembly-module
+        text: instantiate; url: #dom-webassembly-instantiate
+        text: asynchronously compile a webassembly module; url: #asynchronously-compile-a-webassembly-module
+        text: instantiate a promise of a module; url: #instantiate-a-promise-of-a-module
+</pre>
+
+<pre class='link-defaults'>
+spec:infra; type:dfn; text:list
+spec:ecma-262; type:exception; for:ECMAScript; text:Error
+spec:infra; type:dfn; for:set; text:append
+</pre>
+
+This document builds off of the WebAssembly specification [[WEBASSEMBLY]] and the WebAssembly JavaScript embedding [[WASMJS]].
+
+<h2 id="streaming-modules">Streaming module APIs</h2>
+
+<pre class="idl">
+partial namespace WebAssembly {
+  Promise&lt;Module> compileStreaming(Promise&lt;Response> source);
+  Promise&lt;WebAssemblyInstantiatedSource> instantiateStreaming(
+      Promise&lt;Response> source, optional InstanceImportsMap importObject);
+};
+</pre>
+
+<h3 id="streaming-compliaton">Streaming module compilation</h3>
+
+Note: This method was not included in the initial Wasm specification, so developers may need to provide a code path for the case where it is not present.
```

Comment:
s/Wasm/WebAssembly/ . Also, technically this *is* the initial specification, so the comment seems a bit odd now. I'd be fine with just dropping it.

Reply:
I haven't seen any other web specs use language like this; it's sort of implicit in everything that developers have to look at what's supported and do things appropriately. I'd be fine with removing the text wherever it appears.

However, I don't think making this spec in a more official form with Bikeshed really changes anything, making this "the initial specification". The previous one was used for four shipping implementations, so it's as official as anything. It's all evolving over time.

We could include comments like this just when a feature is beginning to ship, and gradually delete them over time, as they become widely shipped. Is that what you're suggesting?

Reply:
Nah, as you say it's weird. I'd just drop it already, but I don't know what others think.

I meant that "technically" this is the initial specification because it is the only one that actually will get official ratification as a standard. Everything before was just design docs, from the POV of view of a standards lawyer that doesn't really count.

Reply:
I would drop it.